includes/admin/views/modals/field-delete.php

```diff
@@ -0,0 +1,35 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	die( 'You are not allowed to call this page directly.' );
+}
+?>
+<div id="wpbdp-field-delete-modal" class="hidden settings-lite-cta">
```

Comment:
Can we make a more generic confirmation modal that we can use for plan deleting too? This looks like it could get out of hand to have to add this much new code for every confirmation.

Reply:
Agreed :)